src/easylink/devtools/implementation_creator.py

```diff
@@ -195,9 +195,17 @@ def register(self) -> None:
                 f"Implementation '{self.implementation_name}' already exists in the registry. "
                 "Overwriting it with the latest data."
             )
+
+        # Handle the fact that developers might be saving to username subdirs
+        image_name = (
```

Comment:
Hacky, but can't really think how else to handle this subdir issue.

Reply:
Aren't we *always* saving to username subdirs? What are the two different cases here?

Reply:
Not if you explicitly pass an output directory into the `create-implementation -o` option. If you _don't_, then the default is at `/mnt/ihme/simulation_science.../<USERNAME>`

Reply:
Ah! I see.